### PR TITLE
Audio Super Resolution ✨ 

### PR DESCRIPTION
**Pull request by RyanMetcalfeInt8**
This integrates Audio Super Resolution into this project.

Note: I've tested that this builds on Ubuntu 24.04

OpenVINO models are pulled from here: https://huggingface.co/Intel/versatile_audio_super_resolution_openvino

### CODE REVIEW

**Comment by RyanMetcalfeInt8** [comment]
> Looks good! Please do a review multiple uses of hardcoded vars for 2 things:
> 
> 1. would/could these become parameters?
> 2. are their values modified or should they have const declaration.

Thanks. I just went through and added 'const' to various hardcoded vars. For values that could potentially be exposed as settable params, I added some additional comments -- but this was only applicable in a couple places. In general, there were many hardcoded values in the original python routines that I ported from.

File: doc/build_doc/linux/README.md [hunk 1]
```diff
@@ -291,115 +291,26 @@ cd openvino-models
...
-
-
-
-7 directories, 74 files
-```
+# clone the HF repo
+git clone https://huggingface.co/Intel/versatile_audio_super_resolution_openvino
 
+# unzip the 'base' set of models into audiosr
+unzip versatile_audio_super_resolution_openvino\versatile_audio_sr_base_openvino_models.zip -d openvino-models\audiosr
```
Comment (gblong1) [inline doc/build_doc/linux/README.md:303]: Slashes are backwards for Linux in this section. Should be / vs current \. Modifying this in the README will make cut-n-paste easier.

**Review by gblong1 — CHANGES_REQUESTED** [review]
Small change needed for the Linux readme build instructions.

File: doc/build_doc/linux/README.md (hunk 1 above)
Comment (RyanMetcalfeInt8) [reply doc/build_doc/linux/README.md]: Ah, thanks -- fixed.

**Review by gblong1 — APPROVED** [review]
LGTM!

**Pull request merged**